plugin.json

```diff
@@ -59,6 +59,20 @@
                 "help_text": "The webhook secret used by the ServiceNow API calls to Mattermost for sending message response.",
                 "regenerate_help_text": "Regenerates the secret for ServiceNow Virtual Agent Plugin. Regenerating this key invalidates any existing key.",
                 "default": ""
+            },
+            {
+                "key": "ChannelCacheSize",
+                "display_name": "DM Channel Cache Size:",
+                "type": "number",
+                "help_text": "The size of the cache that is used to store DM channel IDs.",
+                "default": 10000
+            },
+            {
+                "key": "ChannelCacheTTL",
+                "display_name": "DM Channel Cache Expiry Time:",
+                "type": "number",
+                "help_text": "The expiry time (in minutes) after which the entries in the cache are deleted.",
+                "default": 10
```

Comment:
3/5 that this is not something an admin should configure. How should they know what to use?

Reply:
Maybe we can give suggestions in the help text so that the admin can get some idea about what to use. Or do you prefer it to be a constant value in the code?

Reply:
I'm leaning towards a constant value as admins don't have a way to figure out what impact a "good" value has on their instance. We don't provide any monitoring (which would be a bit too much).